cms/templates/activation_invalid.html

```diff
@@ -17,7 +17,7 @@ <h1 class="page-header">${_("{studio_name} Account Activation").format(studio_na
       <div class="msg">
         <h1 class="title">${_("Your account activation is invalid")}</h1>
         <div class="copy">
-          <p>${_("We're sorry. Something went wrong with your activation. Check to make sure the URL you went to was correct — e-mail programs will sometimes split it into two lines.")}</p>
```

Comment:
pybabel will always produces `\\u2014` (incorrect) rather than `\u2014` (correct) for the character `—` in this string as the `msgid` string, which will make django not translate this string. Furthermore, any non-ascii character or an escaped character will also cause the same problem.